### PR TITLE
Minor updates Graphene python worker README

### PR DESCRIPTION
**Pull request by rranjan3**
Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>

### CODE REVIEW

File: examples/graphene_apps/python_worker/README.md [hunk 1]
```diff
@@ -166,6 +168,16 @@ https://creativecommons.org/licenses/by/4.0/
...
+## FAQs
+
+  1. What to do when I see an error `insmod: ERROR: could not insert module gsgx.ko: File exists` while installing Graphene SGX driver?
+
+     Ignore the error and proceed. You have probably already installed the Graphene SGX driver.
+
+  2. What should I do if I see an error `ModuleNotFoundError: No module named 'docker'`?
+
+     You need to install python paackage for docker using `python3 -m pip install docker`.
+
```
Comment (manojgop) [inline examples/graphene_apps/python_worker/README.md:180]: The FAQ looks out of scope for this ReadMe.md. Also we need to remove gsgx driver and use in kernel driver with FSGSBASE patch soon.

File: examples/graphene_apps/python_worker/README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ https://creativecommons.org/licenses/by/4.0/
 - Before building and running application for Graphene-SGX, we need to install Intel SGX driver and Graphene SGX driver.
 
   - To install Intel SGX driver please refer https://github.com/hyperledger/avalon/blob/master/PREREQUISITES.md#intel-sgx-in-hardware-mode
-  - To install Graphene SGX driver please refer https://graphene.readthedocs.io/en/latest/building.html#install-the-graphene-sgx-driver-not-for-production
+  - Clone the Graphene codebase from [here](https://github.com/oscarlab/graphene.git) if not done already. Then, to install Graphene SGX driver please refer https://graphene.readthedocs.io/en/latest/building.html#install-the-graphene-sgx-driver-not-for-production. Also, note that you might have to install a few other dependencies if you have not built Graphene on this setup. Refer [this](https://graphene.readthedocs.io/en/latest/building.html#required-packages) for assistance.
```
Comment (manojgop) [inline examples/graphene_apps/python_worker/README.md:64]: We are not going to build graphene stand alone. GSC tool will download and build it. Did you face any issue if you didn't install the dependencies ?

File: examples/graphene_apps/python_worker/README.md [hunk 3]
```diff
@@ -130,7 +132,7 @@ https://creativecommons.org/licenses/by/4.0/
      *graphene-python-worker    | Bind to zmq port*
      *graphene-python-worker    | waiting for next request*
 
-  3. To send work orders to python worker we can use Avalon generic client application.
+  3. To send work orders to python worker we can use Avalon generic client application as show [above](https://github.com/hyperledger/avalon/tree/master/examples/graphene_apps/python_worker#test-python-worker-using-avalon).
```
Comment (danintel) [inline examples/graphene_apps/python_worker/README.md:135]: change "show" to "shown"

File: examples/graphene_apps/python_worker/README.md [hunk 4]
```diff
@@ -166,6 +168,16 @@ https://creativecommons.org/licenses/by/4.0/
...
 
+## FAQs
+
+  1. What to do when I see an error `insmod: ERROR: could not insert module gsgx.ko: File exists` while installing Graphene SGX driver?
+
+     Ignore the error and proceed. You have probably already installed the Graphene SGX driver.
+
+  2. What should I do if I see an error `ModuleNotFoundError: No module named 'docker'`?
+
+     You need to install python paackage for docker using `python3 -m pip install docker`.
```
Comment (danintel) [inline examples/graphene_apps/python_worker/README.md:179]: Change paackage to package

File: examples/graphene_apps/python_worker/README.md (hunk 3 above)
Comment (danintel) [inline examples/graphene_apps/python_worker/README.md:135]: Change "show" to "shown"

File: examples/graphene_apps/python_worker/README.md (hunk 2 above)
Comment (danintel) [inline examples/graphene_apps/python_worker/README.md:64]: Change this to 3 separate steps

File: examples/graphene_apps/python_worker/README.md (hunk 1 above)
Comment (rranjan3) [reply examples/graphene_apps/python_worker/README.md:180]: build_gsc_python_worker.sh script does throw such an error. The gsc.py from Graphene call for this dependency.

File: examples/graphene_apps/python_worker/README.md (hunk 2 above)
Comment (rranjan3) [reply examples/graphene_apps/python_worker/README.md:64]: I already had Graphene built earlier on my setup. I might have to do a fresh install to verify it. But, since the driver installation step in the documentation of Graphene mentioned `make`, I suspect the dependencies might throw up issues there.

Comment (manojgop) [reply examples/graphene_apps/python_worker/README.md:64]: we can mention install GSC tool dependent software packages from https://github.com/oscarlab/graphene/blob/master/Documentation/manpages/gsc.rst#software-packages
